lib/process/replaceData.js

```diff
@@ -5,40 +5,43 @@ const { fileExt } = require('./defaults');
 const replaceData = (filePath, fileData, options) => {
   let data;
 
+  const sourceFile = { sourceFile: filePath };
   if (
     options.type === 'js' ||
     (
       options.type === 'auto' &&
       fileExt.js.includes(path.extname(filePath))
     )
   ) {
-    data = rcs.replace.js(fileData, options.parserOptions);
+    // eslint-disable-next-line no-param-reassign
```

Comment:
I think it still works without this line?

Reply:
Will try. I don't know since I can't run test on my side.